packages/teleport-plugin-common/src/utils/ast-utils.ts

```diff
@@ -174,6 +174,19 @@ export const addAttributeToJSXTag = (
   jsxNode.openingElement.attributes.push(attributeDefinition)
 }
 
+export const addRawAttributeToJSXTag = (
+  jsxNode: types.JSXElement,
+  attrName: string,
+  attrValue: UIDLRawValue,
+  t = types
+) => {
+  const attributeDefinition = t.jsxAttribute(
+    t.jsxIdentifier(attrName),
+    t.jsxExpressionContainer(convertValueToLiteral(attrValue.content))
+  )
```

Comment:
Here instead of `convertValueToLiteral` you can do something like this
```js
types.templateLiteral([types.templateElement({ raw: attrValue.content })], [])
```

So, this will generate string template literals. Which helps n better readability with lengthy strings